ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -188,9 +188,9 @@
 {%     endif %}
 {# Phone Interfaces #}
 {%     set phone_interfaces = [] %}
-{%     for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
-{%         if ethernet_interface.phone is arista.avd.defined %}
-{%             do phone_interfaces.append(ethernet_interface) %}
+{%     for interface in ethernet_interfaces + port_channel_interfaces | arista.avd.natural_sort('name') %}
```

Comment:
You may need some parantheseses here to get the total list before running the filter. I think the current logic would only apply the sort filter to port_channel_interface.
And btw since you also rely on the filter to detect missing vars, we should probably do:
```suggestion
{%     for interface in ethernet_interfaces | arista.avd.natural_sort('name') + port_channel_interfaces | arista.avd.natural_sort('name') %}
```